crates/matrix-sdk-ui/tests/integration/timeline/echo.rs

```diff
@@ -162,7 +162,7 @@ async fn retry_failed() {
     });
 
     // Sending fails, the mock server has no matching route yet
-    assert_next_matches!(timeline_stream, VectorDiff::Set { index: 0, value } => {
+    assert_matches!(timeline_stream.next().await, Some(VectorDiff::Set { index: 0, value }) => {
```

Comment:
Did you revert because `assert_next_matches` does a `now_or_never` instead of await for the value?

Reply:
Yes.